frontend/src/apidefinition.ts

```diff
@@ -17,20 +17,33 @@ export interface Event {
   edges: Array<Uuid>
 }
 
+export interface StringCompare {
```

Comment:
Unnecessary export.

Reply:
Its used in exported methods, and the type is part of the eiffelvis api; you will have to use it at some point, but at the moment we simply don't name the type it in code yet.

Reply:
But those exported methods are local in the same file as the interface. Just exporting because
> you will use it at some point

feels redundant.

Reply:
Returning private types (for example in `string_compare_default`) makes no sense to me.
Same thing for other exported type definitions that include `StringCompare` (for example `SourceHost`), all fields are public; but then StringCompare would be private? how can you logically access the fields then?

Enlighten me why it would make sense :smile: